jpo-geojsonconverter/src/main/java/us/dot/its/jpo/geojsonconverter/pojos/tim/PathNodeData.java

```diff
@@ -0,0 +1,16 @@
+package us.dot.its.jpo.geojsonconverter.pojos.tim;
+
+import java.util.List;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+/**
+ * Data structure to hold coordinate and offset information for TIM path processing.
+ */
+@Data
+@AllArgsConstructor
+public class PathNodeData {
+    private final List<Double> coordinates;
+    private final Long dwithOffset;
```

Comment:
question: why the "d" prefix on this and the below property?

suggestion: if this is "width", we should adjust spelling
```suggestion
    private final Long dwidthOffset;
```